util/testleak/add-leaktest.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+#
+# Usage: add-leaktest.sh pkg/*_test.go
+
+set -eu
+
+sed -i'~' -e '
```

Comment:
Have u tested in `MAC OSX`, i remember that the `sed` command is not the same with `*nix`.